src/main/scala/sbtlicensereport/SbtLicenseReport.scala

```diff
@@ -15,6 +15,7 @@ object SbtLicenseReport extends AutoPlugin {
     def LicenseCategory = sbtlicensereport.license.LicenseCategory
     type TargetLanguage = sbtlicensereport.license.TargetLanguage
     type LicenseReportConfiguration = sbtlicensereport.license.LicenseReportConfiguration
+    type DepModuleInfo = sbtlicensereport.license.DepModuleInfo
```

Comment:
I added this with the intention of it being visible within `autoImport` but it doesn't seem to be working (see https://github.com/sbt/sbt-license-report/pull/60/files#r1169846526).

Any idea why?

Reply:
You probably need both type name and term name (`val`) for the case class companion to be forwarded.

Reply:
Perfect that solved the issue, resolving conversation.